test/Mvc6Framework45.FunctionalTests/Models/AccountViewModels.cs

```diff
@@ -4,6 +4,7 @@
 using System.Threading.Tasks;
 using System.ComponentModel.DataAnnotations;
 using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.Rendering;
```

Comment:
I will address the comments and see if it compiles fine with both being there. If that's the case, I will open a bug. Thanks!